yarn/pom.xml

```diff
@@ -147,20 +147,34 @@
       </exclusions>
       <scope>test</scope>
      </dependency>
+
+     <!--
+       Jersey 1 dependencies only required for YARN integration testing. Creating a YARN cluster
+       in the JVM requires starting a Jersey 1-based web application.
+     -->
      <dependency>
        <groupId>com.sun.jersey</groupId>
        <artifactId>jersey-core</artifactId>
        <scope>test</scope>
+       <version>1.9</version>
```

Comment:
One last finishing touch .. you could define (in just this pom) a property for Jersey 1.x versions to make sure they're modified together.